data/remnant/remnant outfits.txt

```diff
@@ -276,6 +277,136 @@ outfit "EMP Torpedo Bay"
 		"missile strength" 80
 	description "Electromagnetic pulse weapons were developed by the Remnant during the early days of their colony, when they were living in fear that the Alphas would overrun human space and expand beyond it."
 
+
+
+outfit "Teciimach Canister"
+	plural "Teciimach Canisters"
+	category "Ammunition"
+	licenses
+		Remnant
+	cost 5500
+	thumbnail "outfit/emp torpedo"
+	"mass" 1
+	"teciimach canister capacity" -1
+	description "This canister contains everything required to assemble a Teciimach torpedo. These weapons emit strong electromagnetic pulses which interfere with a ship's generators and neutralize a portion of their energy."
+	description `	Unlike the previous generation of "EMP torpedoes," the Teciimach torpedoes actively shape their emission pattern to avoid impacting their allies.`
+
+outfit "Teciimach Canister Rack"
+	plural "Teciimach Canister Racks"
+	category "Ammunition"
+	licenses
+		Remnant
+	cost 98000
+	thumbnail "outfit/emp rack"
+	"mass" 5
+	"outfit space" -9
+	"energy capacity" 500
+	"teciimach canister capacity" 9
```

Comment:
5 mass and 9 outfit space would imply a capacity of 4.
```suggestion
	"teciimach canister capacity" 4
```

Reply:
No, the masses are not cumulative with outfit space. This is deliberate.